.github/workflows/build-deploy-frontend.yml

```diff
@@ -9,11 +9,6 @@ on:
         type: choice
         options:
           - dev
-          - dev2
```

Comment:
Awesome!  This is great.  Per our discussion, I do not think we need so many lower or development environments.  If we go to production we should only have approximately 3 (dev, demo or stage and production).  Will you also be updating the [documentation](https://docs.google.com/document/d/1MHpk3CTb7m0W-bINZyrfb_k0L3poWlYghkVBP_NqrSs/edit?tab=t.0#heading=h.12vtxv6luwj4) to show the changes?

Reply:
You can also remove the `ops/terraform/outputs.tf` file since it is blank.

Reply:
very true! I think that might have gotten created back during the POC phase and never ended up getting used.